client/components/domains/domain-search-results/index.jsx

```diff
@@ -80,6 +80,7 @@ class DomainSearchResults extends React.Component {
 			MAPPABLE,
 			MAPPED,
 			TLD_NOT_SUPPORTED,
+			TLD_NOT_SUPPORTED_AND_NAME_NOT_AVAILABLE,
```

Comment:
I mentioned this in comments on the back-end patch as well, and I know this is the nitpickiest of nitpicks, but could we please change this to `TLD_NOT_SUPPORTED_AND_DOMAIN_NOT_AVAILABLE`, just so it's perfectly clear that we're talking about a domain name here.